test/emit-declaration/emit-declaration.test.js

```diff
@@ -20,17 +20,6 @@ test('Can emit valid declaration with sourceMap', async () => {
     'utf-8'
   );
 
-  expect(declaration).toMatchInlineSnapshot(`
-"// This file is automatically generated.
-// Please do not change this file!
-interface CssExports {
-  'otherClass': string;
-  'someClass': string;
-  'validClass': string;
-}
-declare var cssExports: CssExports;
-export = cssExports;
-"
-`);
+  expect(declaration).toMatchSnapshot();
```

Comment:
Looks like the fix has broken this test.

Reply:
I just switched that one to a snapshot instead of an inline snapshot. I’m not sure why it was inline before vs. the external snapshot for the other case.